packages/webapp/__tests__/PopularPage.tsx

```diff
@@ -104,22 +103,6 @@ const renderComponent = (
   );
 };
 
-it('should request user feed', async () => {
```

Comment:
Removal of this test case was intentional, as we will now never request for personalized feeds on the popular page with the `my feed` feature being turned on for everyone.